getting-started/readme.adoc

```diff
@@ -44,6 +44,10 @@ Verify the version:
     $ minikube version
     minikube version: v0.23.0
 +
+If `minikube` has already been installed earlier, you will also want to delete the previous minikube ISO:
```

Comment:
The flow of the instructions at https://github.com/mdelagrange/aws-workshop-for-kubernetes/blob/b9bf5cf9ef53513ad0dec22fc7b518171e7bc1fd/getting-started/readme.adoc#setup-on-macos does not seem logical.

. Install minikube
. If already installed, then update
. If error then give a `sudo cp` command
. If already installed, then delete the ISO

Can you please look through the instructions and keep them simple?

Reply:
Sure, I'll give it another look.